### PR TITLE
Update Polish localization of README.md

### PR DESCRIPTION
**Pull request by mfilocha**
Synchronize Polish localization of the main README file up to 1eddc8bfedfbf9e9253600f36789333281f515a6.

### CODE REVIEW

**Comment by mfilocha** [comment]
Adding @nvtkaszpir and @kpucynski for language quality review.

File: README-pl.md [hunk 1]
```diff
@@ -1,90 +1,152 @@
...
+Przed rozpoczęciem zainstaluj niezbędne zależności. Sklonuj repozytorium i przejdź do odpowiedniego katalogu:
 
-Zalecaną metodą uruchomienia serwisu internetowego Kubernetesa lokalnie jest użycie specjalnego obrazu [Dockera](https://docker.com), który zawiera generator stron statycznych [Hugo](https://gohugo.io).
+```
+git clone https://github.com/kubernetes/website.git
+cd website
+```
 
-> Użytkownicy Windows będą potrzebowali dodatkowych narzędzi, które mogą zainstalować przy pomocy [Chocolatey](https://chocolatey.org).
+Strona Kubernetes używa [Docsy Hugo theme](https://github.com/google/docsy#readme). Nawet jeśli planujesz uruchomić serwis w środowisku kontenerowym, zalecamy pobranie podmodułów i innych zależności za pomocą polecenia:
```
Comment (kpucynski) [inline README-pl.md:30]:
```suggestion
Strona Kubernetesa używa [Docsy Hugo theme](https://github.com/google/docsy#readme). Nawet jeśli planujesz uruchomić serwis w środowisku kontenerowym, zalecamy pobranie podmodułów i innych zależności za pomocą polecenia:
```

File: README-pl.md [hunk 2]
```diff
@@ -1,90 +1,152 @@
...
 
-```bash
-choco install make
+```
+# pull in the Docsy submodule
+git submodule update --init --recursive --depth 1
 ```
 
-> Jeśli wolisz uruchomić serwis lokalnie bez Dockera, przeczytaj [jak uruchomić serwis lokalnie przy pomocy Hugo](#jak-uruchomić-lokalną-kopię-strony-przy-pomocy-hugo) poniżej.
+## Uruchomienie serwisu poprzez kontener
```
Comment (kpucynski) [inline README-pl.md:37]:
```suggestion
## Uruchomienie serwisu w kontenerze
```

File: README-pl.md [hunk 3]
```diff
@@ -1,90 +1,152 @@
...
 
-Jeśli [zainstalowałeś i uruchomiłeś](https://www.docker.com/get-started) już Dockera, zbuduj obraz `kubernetes-hugo` lokalnie:
+Aby zbudować serwis wewnątrz środowiska kontenerowego, wykonaj następujące polecenia aby zbudować obraz i go uruchomić:
```
Comment (kpucynski) [inline README-pl.md:39]:
```suggestion
Aby zbudować i uruchomić serwis wewnątrz środowiska kontenerowego, wykonaj następujące polecenia:
```

File: README-pl.md [hunk 4]
```diff
@@ -1,90 +1,152 @@
...
+sudo mv limit.maxfiles.plist /Library/LaunchDaemons
+sudo mv limit.maxproc.plist /Library/LaunchDaemons
+
+sudo chown root:wheel /Library/LaunchDaemons/limit.maxfiles.plist
+sudo chown root:wheel /Library/LaunchDaemons/limit.maxproc.plist
+
+sudo launchctl load -w /Library/LaunchDaemons/limit.maxfiles.plist
+```
+
+Powinno działać zarówno dla wersji Catalina i Mojave macOS-a.
```
Comment (kpucynski) [inline README-pl.md:105]:
```suggestion
Przedstawiony sposób powinien działać dla MacOS w wersji Catalina i Mojave.
```

File: README-pl.md [hunk 5]
```diff
@@ -1,90 +1,152 @@
...
+
+
+# Zaangażowanie w prace SIG Docs
+
+O społeczności SIG Docs i terminach spotkań dowiesz się przez [jej stronę](https://github.com/kubernetes/community/tree/master/sig-docs#meetings).
```
Comment (kpucynski) [inline README-pl.md:110]:
```suggestion
O społeczności SIG Docs i terminach spotkań dowiesz z [jej strony](https://github.com/kubernetes/community/tree/master/sig-docs#meetings).
```

**Review by kpucynski — CHANGES_REQUESTED** [review]
minor things

**Review by kpucynski — APPROVED** [review]
/lgtm

**Comment by sftim** [comment]
/approve
Thanks @mfilocha

**Pull request merged**